iceoryx_posh/include/iceoryx_posh/internal/popo/ports/server_port_roudi.hpp

```diff
@@ -62,6 +65,13 @@ class ServerPortRouDi : public BasePort
     const MemberType_t* getMembers() const noexcept;
     MemberType_t* getMembers() noexcept;
 
+    void handleCaProProtocolViolation(const capro::CaproMessageType messageType) const noexcept;
```

Comment:
I think this can be made static - I suggestion from clang-tidy.

Reply:
I extended the debug output to make it more similar to `ClientPortUser` which requires an access to members and therefore it must remain a method.